src/models/galp4/external-overview.md

```diff
@@ -35,6 +35,8 @@ The Galago Pro has the following actions available using the Fn and Function key
 |![Fn-F11](./img/fn-f11.png)|Fn+F11  |Toggle airplane mode               |
 |![Fn-F12](./img/fn-f12.png)|Fn+F12  |Suspend                            |
 |![Fn-F12](./img/fn-dia.jpg)|Fn+`    |Play/Pause                         |
+|Ins                        |Fn+Ins  |Enable Numpad                      |  
```

Comment:
Is the numpad a toggle (press/release the shortcut to turn it on, press/release again to turn it off), or do you have to hold down Fn+Ins while using the numpad?

Reply:
It is a toggle similar to the other functions.

Reply:
Cool, I'd like to call it a toggle to match the other lines and clarify that it also disables and doesn't just enabled, then, if that's okay. Made the change with 4e8a06b.

Reply:
Do we want to change the verbiage to Toggle Numpad thinking about it now.

Reply:
Yep, that's exactly what I did with that commit.